### DISCUSSION TITLE
Progress on initial capability

### DISCUSSION THREAD

**Discussion by logantm2** [logantm2/cavernus#2] [General]
- [x] Theory document
- [x] Classes for body forces
- [x] Classes for initial conditions
- [x] Classes for boundary conditions
- [x] Classes for elasticity tensors
- [x] "K" (total strain stiffness matrix) integrator
- [x] "G" (creep strain matrix) integrator
- [x] "F" (creep strain rate) integrator
- [x] Explicit operator
~~- [ ] Implicit solver~~ This can be added later on

**Comment by logantm2** [comment]
@passthebasalt I think this is finally in a state where we can start doing some science with it. I haven't put a ton of time into getting realistic parameters, but I put some fake numbers in and was able to get sensible output. Unlike the code from the Kumar paper, the output field is actually deformed according to the displacement solved by the code. See below:

![image](https://github.com/logantm2/cavernus/assets/53028846/a33fac2a-0de1-427d-937d-79b677552970)

Wanna set up a call or something soon to discuss? I can go over how the code works and what else could be done.

**Comment by passthebasalt** [comment]
Yes let’s schedule a call to talk about this! I’m pretty booked tomorrow but I’m around any time Friday.


Regards,



Justin Vinitsky Delgado

PhD Student (Level I) | CUNY Graduate Center
Grade B Laboratory Instructor  | Brooklyn College


Errors and brevity due to mobile.
________________________________
From: logantm2 ***@***.***>
Sent: Wednesday, July 19, 2023 6:28:02 PM
To: logantm2/cavernus ***@***.***>
Cc: Justin Vinitsky Delgado ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL]Re: [logantm2/cavernus] Progress on initial capability (Discussion #2)

***ATTENTION: This email came from an external source. Do not open attachments or click on links from unknown senders or unexpected emails.***


@passthebasalt<https://github.com/passthebasalt> I think this is finally in a state where we can start doing some science with it. I haven't put a ton of time into getting realistic parameters, but I put some fake numbers in and was able to get sensible output. Unlike the code from the Kumar paper, the output field is actually deformed according to the displacement solved by the code. See below:

[image]<https://user-images.githubusercontent.com/53028846/254725299-a33fac2a-0de1-427d-937d-79b677552970.png>

Wanna set up a call or something soon to discuss? I can go over how the code works and what else could be done.

**Reply by logantm2** [reply]
Sounds good. Maybe 2 your time on Friday?

**Comment by passthebasalt** [comment]
Yea let’s do it


Regards,



Justin Vinitsky Delgado

PhD Student (Level I) | CUNY Graduate Center
Grade B Laboratory Instructor  | Brooklyn College


Errors and brevity due to mobile.
________________________________
From: logantm2 ***@***.***>
Sent: Thursday, July 20, 2023 4:38:59 PM
To: logantm2/cavernus ***@***.***>
Cc: Justin Vinitsky Delgado ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL]Re: [logantm2/cavernus] Progress on initial capability (Discussion #2)

***ATTENTION: This email came from an external source. Do not open attachments or click on links from unknown senders or unexpected emails.***


Sounds good. Maybe 2 your time on Friday?

**Comment by passthebasalt** [comment]
Topic: Salty meeting
Time: Jul 21, 2023 02:00 PM Eastern Time (US and Canada)

Join Zoom Meeting
https://us05web.zoom.us/j/84973500524?pwd=AmBPt2XM257pDeWVhSVGFrqbsWGbgb.1

Meeting ID: 849 7350 0524
Passcode: d3tMzM
